.circleci/deploy_to_kubernetes

```diff
@@ -32,3 +32,12 @@ cat "$NAMESPACE_DIR/deployment.yml" | \
     --filename=/dev/stdin \
     --filename="$NAMESPACE_DIR/service.yml" \
     --filename="$NAMESPACE_DIR/ingress.yml"
+
+# Worker queue deployment
+if [ -f "$NAMESPACE_DIR/queue_worker_deployment.yml" ] ; then
```

Comment:
Is there any chance this file is missing?

Reply:
when doing local deployments. The cloud platform amq-broker won't be accessible locally

Reply:
Sorry, I don't understand. The way I read this means we don't want to deploy worker pods in the "development" `$NAMESPACE_DIR`, which would mean we have a half-functional environment, right? (Uploads will not work.)

Since the development "namespace" starts a RabbitMQ pod, it sounds to me that we should still have a deployment file with the "local" RabbitMQ as the configuration for the AMQP broker. What do you think?

Reply:
I think you are right. I will add this in https://github.com/ministryofjustice/laa-legal-adviser-api/pull/118